experimenter/experimenter/experiments/tests/api/v5/test_serializers/test_nimbus_experiment_serializer.py

```diff
@@ -953,13 +955,23 @@ def test_update_publish_status_to_approved_invokes_push_task(self, application):
         )
         self.assertTrue(serializer.is_valid())
 
-        experiment = serializer.save()
+        expected_collection = experiment.application_config.get_kinto_collection_for(experiment)
+
+        with patch.object(
+            experiment.application_config,
+            "get_kinto_collection_for",
+            wraps=experiment.application_config.get_kinto_collection_for,
+        ) as get_kinto_collection_for:
+            experiment = serializer.save()
+            get_kinto_collection_for.assert_called_once_with(experiment)
```

Comment:
Wait why do we have to mock and assert on this call here?  Isn't it sufficient just to check which collection is called by the kinto client later?